x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_panel.tsx

```diff
@@ -105,5 +116,61 @@ const SolutionGroupedNavPanelComponent: React.FC<SolutionGroupedNavPanelProps> =
     </>
   );
 };
+export const SolutionNavPanel = React.memo(SolutionNavPanelComponent);
+
+const SolutionNavPanelCategories: React.FC<SolutionNavPanelCategoriesProps> = ({
+  categories,
+  items,
+  onClose,
+}) => {
+  const itemsMap = new Map(items.map((item) => [item.id, item]));
+
+  return (
+    <>
+      {categories.map(({ label, linkIds }) => {
+        const links = linkIds.reduce<DefaultSideNavItem[]>((acc, linkId) => {
+          const link = itemsMap.get(linkId);
+          if (link) {
+            acc.push(link);
+          }
+          return acc;
+        }, []);
+
+        return (
+          <Fragment key={label}>
+            <EuiTitle size="xxxs">
+              <h2>{label}</h2>
+            </EuiTitle>
+            <EuiHorizontalRule margin="s" />
+            <SolutionNavPanelItems items={links} onClose={onClose} />
+            <EuiSpacer size="l" />
+          </Fragment>
+        );
+      })}
+    </>
+  );
+};
 
-export const SolutionGroupedNavPanel = React.memo(SolutionGroupedNavPanelComponent);
+const SolutionNavPanelItems: React.FC<SolutionNavPanelItemsProps> = ({ items, onClose }) => (
+  <>
+    {items.map(({ id, href, onClick, label, description }) => (
+      <Fragment key={id}>
+        <EuiDescriptionListTitle>
+          <a
```

Comment:
Is there any particular reason we use <a> here?

Reply:
Just because it has to take the styles from `EuiDescriptionListTitle`, using `EuiLink` inside would add some more styles that are not needed.